### ISSUE TITLE
Contact on FAQs should be a `h2`

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#425]
![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiMTRiNDc5ZjMtZWM2YS00ZDBiLWI5NGUtNmZlZmUzMzc4NWUzIiwiZGVwbG95X2lkIjoiNjExZTg5OGE5NDQ4N2IwMDA3NTUxNTAzIiwiaWQiOiI2NzJjYzlhNi1kNzk2LTQ5MTctOGFjMC05ZGI5NzU0NjU3ZDIifQ==)

### Description of issue
Contact is a `h3` which makes it seem like its subcontent to accessibility.

### Expected result
Contact is a `h2`

```
Path:      /faqs/
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  1792 x 1009 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2Ffaqs%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1792x1009&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/faqs/)

**Comment by marcusrelacion** [comment]
@joshkimux this has been fixed.
